pages/content/amp-dev/documentation/guides-and-tutorials/develop/style_and_layout/index.md

```diff
@@ -78,18 +70,18 @@ using common CSS properties. For example:
 </body>
 [/sourcecode]
 
-[tip type="important"]
-**IMPORTANT –**: Check that your styles are supported in AMP; some styles aren't for performance reasons (see also [Supported CSS](style_pages.md)).
-[/tip]
-
-If needed, AMP also allows inline styles. For example:
+If needed, AMP allows inline styles:
 
 [sourcecode:html]
 <body>
   <p style="color:pink;margin-left:30px;">Hello, Kitty.</p>
 </body>
 [/sourcecode]
 
+[tip type="important"]
+  Inline styles detract from the total 50,000 byte limit. 
```

Comment:
If the limit isn't actually reduced, `count toward` may be preferable to `detract from`.

A similar scenario occurs [in this document](https://docs.microsoft.com/en-us/microsoftteams/limits-specifications-teams), search `toward` on page.

Reply:
Thanks, great suggestion, I'll update the wording [here](https://github.com/ampproject/amp.dev/pull/3302) to match as well!